### PR TITLE
[MAINT + ENH, MRG] Fix identity transforms

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes https://github.com/mne-tools/mne-python/issues/9680

I guess there were more than a few examples where people assumed identity transforms...

### CODE REVIEW

File: examples/forward/left_cerebellum_volume_source.py [hunk 1]
```diff
@@ -46,10 +49,11 @@
 # %%
 # Plot the positions of each source space
 
-fig = mne.viz.plot_alignment(subject=subject, subjects_dir=subjects_dir,
+fig = mne.viz.plot_alignment(trans=trans, subject=subject,
+                             subjects_dir=subjects_dir,
                              surfaces='white', coord_frame='head',
                              src=src)
```
Comment (larsoner) [inline examples/forward/left_cerebellum_volume_source.py:55]: Rather than pass+use an incorrect trans here, I'd rather do `coord_frame='mri'`

**Review by larsoner — COMMENTED** [review]
Other than an idea for one simplification LGTM

File: examples/forward/left_cerebellum_volume_source.py (hunk 1 above)
Comment (alexrockhill) [reply examples/forward/left_cerebellum_volume_source.py]: After I had adjusted the roll and everything ;)

**Comment by alexrockhill** [comment]
Ok should be good to go here, I forgot to iterate over views which we actually used in the ECoG tutorial, my mistake!

**Comment by larsoner** [comment]
Thanks @alexrockhill !

**Pull request merged**